src/components/binding/binding-panel.js

```diff
@@ -213,13 +187,51 @@ genotet.BindingPanel.prototype.updateTracks = function() {
       this.container.find('#genes #track-' + index).remove();
     }
   }
-  this.data.tracks.forEach(function(track, index) {
-    var ui = this.container.find('#genes #track-' + index);
-    if (!ui.length) {
-      this.addTrack_();
-    }
-    this.selectGenes_[index].val(track.fileName).trigger('change');
-  }, this);
+
+  // Load binding list and update panel tracks.
+  var params = {
+    type: 'list-binding'
+  };
+  $.get(genotet.data.serverURL, params, function(data) {
```

Comment:
I didn't notice you are trying to get under panel. Unfortunately panel is not supposed to send ajax request. Yes you need to send a signal to view and let view pass that signal to loader.

All requests should be sent from the loader. Otherwise you are making a mess of functionality split between components.